### PR TITLE
Add manifests for installation

### PR DESCRIPTION
**Pull request by deads2k**
Adds manifests for installation

### CODE REVIEW

File: manifests/0000_12_etcd-operator_01_config.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: etcds.operator.openshift.io
+spec:
+  scope: Cluster
+  group: operator.openshift.io
```
Comment (hexfusion) [inline manifests/0000_12_etcd-operator_01_config.crd.yaml]: I just wonder if we should make this EtcdCluster isn't Cluster `clusters.cluster.k8s.io`?

Comment (deads2k) [reply manifests/0000_12_etcd-operator_01_config.crd.yaml]: > I just wonder if we should make this EtcdCluster isn't Cluster `clusters.cluster.k8s.io`?

`oc get etcd.operator.openshift.io`

**Comment by deads2k** [comment]
`oc apply -f ../etcd-man/manifests/`kind of works

**Comment by deads2k** [comment]
update the deployment to point to the images you built and pushed.

**Pull request merged**